db/migrate/20220620010101_create_custom_translations.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class CreateCustomTranslations < ActiveRecord::Migration[5.2]
+  def change
+    create_table :custom_translations do |t|
```

Comment:
I simply carried over the existing naming convention used by the plugin.  I was forced to change the type as it is a reserved word.

Reply:
It might be a good opportunity to conform with the same attribute names core uses. At the moment it's just a global string replacement. In the future it would mean a migration.

Reply:
that's fine, I can update

Reply:
I've done my best here, there are still more out of scope of translations, but hope this takes us most of the way.